### ISSUE TITLE
[zopengl]  error: unable to resolve comptime value

### ISSUE THREAD

**Issue by Aronicu** [zig-gamedev/zig-gamedev#695]
Zig version: 0.14.0-dev.1472+3929cac15
I just copied the minimal glfw and gl in samples and it gives a compiler error

```
libs\zopengl\src\zopengl.zig:1241:21: error: unable to resolve comptime value
    @export(bindings.cullFace, .{ .name = "glCullFace", .linkage = linkage });
            ~~~~~~~~^~~~~~~~~
libs\zopengl\src\zopengl.zig:1241:21: note: export target must be comptime-known
```

**Comment by hazeycode** [comment]
I'll look into this. In the meantime, try just deleting the exports. They are not required unless you want to use the bindings over C ABI.

**Comment by hazeycode** [comment]
https://github.com/ziglang/zig/pull/19630 related ?

**Comment by CoffeeImpliesCode** [comment]
Replacing `@export(bindings...` with `@export(&bindings...` in every occurrence starting with
https://github.com/zig-gamedev/zig-gamedev/blob/b46101a543fe21a63c0c045751292cc034aecd96/libs/zopengl/src/zopengl.zig#L1241-L1244
up to line 1808 seems to fix this. After changing that all tests/benchmarks work and minimal_glfw_gl as well as minimal_zgui_glfw_gl, although minimal_sdl_gl segfaults for me in sdl.init but I think that is unrelated.
Tested with 0.14.0-dev.1509+fb81522e0

**Comment by hazeycode** [comment]
> Relacing @export(bindings... with @export(&bindings...
Thanks, i can confirm that this works for me also. We should apply this change on the zig-0.14.0 branfch

> although minimal_sdl_gl segfaults for me in sdl.init
Yeah there seems to have been a regression running that sample app in linux. This is present on main branch too. I think it is my fault, I will look into it.

**Comment by hazeycode** [comment]
Fixed on zig-0.14.0 branch here https://github.com/zig-gamedev/zig-gamedev/commit/975483f990c2150dea57c1d9c8e6489283e13270
Thanks @CoffeeImpliesCode
